src/Core/src/Platform/Windows/ViewExtensions.cs

```diff
@@ -27,8 +27,15 @@ public static void TryMoveFocus(this FrameworkElement platformView, FocusNavigat
 				FocusManager.TryMoveFocus(direction, new FindNextElementOptions { SearchRoot = elem });
 		}
 
-		public static void UpdateIsEnabled(this FrameworkElement platformView, IView view) =>
-			(platformView as Control)?.UpdateIsEnabled(view.IsEnabled);
+		public static void UpdateIsEnabled(this FrameworkElement platformView, IView view)
+		{
+			if (platformView is Control control)
+				control.UpdateIsEnabled(view);
```

Comment:
We can probably change this to `control.IsEnabled = view.GetIsEnabled()` and avoid creating another extension method.